### ISSUE TITLE
Web component and React: Local modal - Placeholder (Search by location or language)

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4802]
## Detailed description

Actual Result:
Foreground:#AFADAD
Background:#F4F4F4

The contrast ratio is: 2.0:1
Text failed at Level AA

Expected Result:
Provide below contrast ratio.
Foreground:#7F7E7E
Background:#F4F4F4

The contrast ratio is:3.7:1
Text passed at Level AA
